github/helpers/githubSDKclass.ts

```diff
@@ -0,0 +1,59 @@
+import { IHttp, HttpStatusCode, IRead } from "@rocket.chat/apps-engine/definition/accessors";
+import { ModalsEnum } from "../enum/Modals";
+
+class GitHubApi {
+    private http: IHttp;
+    private BaseHost: string;
+    private BaseApiHost: string;
+    private accessToken: String;
+
+    constructor(http: IHttp, accessToken: String, BaseHost: string, BaseApiHost: string) {
+        this.http = http;
+        this.accessToken = accessToken;
+        this.BaseApiHost = BaseApiHost;
+        this.BaseHost = BaseHost;
+    }
+
+    private async getRequest(url: string): Promise<any> {
+        const response = await this.http.get(url, {
+            headers: {
+                Authorization: `token ${this.accessToken}`,
+                "Content-Type": "application/json",
+            },
+        });
+
+        if (!response.statusCode.toString().startsWith("2")) {
+            throw response;
+        }
```

Comment:
try response.ok() ..probably is available

Reply:
\

Reply:
Sorry that isn't available 
![image](https://github.com/RocketChat/Apps.Github22/assets/99081689/ae64e042-3f41-48da-9224-ee806e16e28e)